src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -2687,6 +2731,65 @@ public async Task OpenApi_GlobalEntityRestPath(bool globalRestEnabled, bool expe
             }
         }
 
+        /// <summary>
+        /// Ensures that the health check endpoint at the app root (/) returns a 200 OK status code
+        /// and the expected JSON response body.
+        /// </summary>
+        /// <returns></returns>
+        [TestMethod]
+        [TestCategory(TestCategory.MSSQL)]
+        public async Task HealthEndpoint_ValidateContents()
+        {
+            // At least one entity is required in the runtime config for the engine to start.
+            // Even though this entity is not under test, it must be supplied to the config
+            // file creation function.
+            Entity requiredEntity = new(
+                Source: new("books", EntitySourceType.Table, null, null),
+                Rest: new(Enabled: false),
+                GraphQL: new("book", "books"),
+                Permissions: new[] { GetMinimalPermissionConfig(AuthorizationResolver.ROLE_ANONYMOUS) },
+                Relationships: null,
+                Mappings: null);
+
+            Dictionary<string, Entity> entityMap = new()
+            {
+                { "Book", requiredEntity }
+            };
+
+            CreateCustomConfigFile(globalRestEnabled: true, entityMap);
+
+            string[] args = new[]
+            {
+                $"--ConfigFileName={CUSTOM_CONFIG_FILENAME}"
+            };
+
+            using TestServer server = new(Program.CreateWebHostBuilder(args));
+            using HttpClient client = server.CreateClient();
+            // Setup and send GET request to root path.
+            HttpRequestMessage getHealthEndpointContents = new(HttpMethod.Get, $"/");
+            HttpResponseMessage response = await client.SendAsync(getHealthEndpointContents);
+
+            // Validate response
+            // Process response body
+            string responseBody = await response.Content.ReadAsStringAsync();
+            Dictionary<string, JsonElement> responseProperties = JsonSerializer.Deserialize<Dictionary<string, JsonElement>>(responseBody);
+
+            // Validate response body
+            Assert.AreEqual(HttpStatusCode.OK, response.StatusCode);
+
+            Assert.IsTrue(
+                condition: responseProperties.TryGetValue(key: "status", out JsonElement statusValue) && statusValue.ToString() == "Healthy",
+                message: "Expected endpoint to be healthy.");
+
+            Assert.IsTrue(
```

Comment:
nit: some of these lines are a bit long, breaking up the conditional might help readability (i just hate scrolling to the right in the editor :)

Reply:
Updated these to be broken up into if statements for the "TryGetValue" which then allowed me to break the conditions up to new lines and be more readable.

Reply:
![image](https://github.com/Azure/data-api-builder/assets/6414189/5fb7e49e-a912-4de9-9821-fdb23505fc14)